oneanddone/tests/functional/pages/tasks/whats_next.py

```diff
@@ -6,6 +6,4 @@
 
 class WhatsNextPage(Base):
```

Comment:
We do end up at that page in one of the tests, but I guess there are no assertions about it, otherwise it would have properties. I think before we were just asserting that we were on the correct page, which was done via page titles, but I removed those. I realize now that that may result in lost coverage. I'll take a look.